AdobeStockAsset/Test/Api/AssetRepository/DeleteTest.php

```diff
@@ -68,17 +115,22 @@ private function deleteAsset(int $assetId)
                 'httpMethod'   => Request::HTTP_METHOD_DELETE,
             ],
             'soap' => [
-                'service'        => self::SERVICE_NAME,
+                'service' => self::SERVICE_NAME,
                 'serviceVersion' => self::SERVICE_VERSION,
-                'opertion'       => self::SERVICE_NAME . 'DeleteById',
+                'operation' => self::SERVICE_NAME . self::SERVICE_OPERATION
```

Comment:
I think it was easier to read with the indentation the way it was

Reply:
Hi @diazwatson 
Sure it was easier to read with indentation.
But we have a few tests and few arrays in the tests which do not contain indentations and I think that we need to use a general approach for arrays code style for this project.
The `GetByIdTest` and `SaveTest` do not contain indentations for `$serviceInfo` array and I think if we apply the required changes here where we need to add the indentation the way like it was it will lead to different code styles for arrays for this project.
My point here is to use one approach for arrays code style.
What the general code style for arrays in this project?
@sivaschenko, can you, please, help us with this question?
Thank you!